### DISCUSSION TITLE
blog/cert-manager-spire-istio/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/rootsongjc.github.io#324] [Announcements]
# blog/cert-manager-spire-istio/

本文介绍如何集成 SPIRE 和使用 cert-manager 实现细粒度的证书管理以及证书轮换。

https://jimmysong.io/blog/cert-manager-spire-istio/

**Comment by wangyushuai** [comment]
证书不是 SpireServer签发的吗？轮转也有Spire管理起来了。 那是不是和Istiod已经没有关系了，还需要cert_manger和 istid参与进来的原因是什么？

**Reply by rootsongjc** [reply]
所有证书是通过cert-manager来签发的。

**Comment by wangyushuai** [comment]
cert_manager给 Spired了一个签发了根证书吗？

**Reply by rootsongjc** [reply]
是的

**Comment by wangyushuai** [comment]
Spire证书是 SpireServer来负责轮转的吗？  grpc反向推送新的证书？

**Reply by rootsongjc** [reply]
SPIRE的证书是cert-manager来轮转的
